js/main.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var fragment = document.createDocumentFragment();
+var MockData = {
+  COMMENTS_MESSAGES: [
+    'Всё отлично!',
+    'В целом всё неплохо. Но не всё.',
+    'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.',
+    'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.',
+    'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.',
+    'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!'
+  ],
+  COMMENTS_NAMES: [
+    'Вася',
+    'Петя',
+    'Коля',
+    'Василиса',
+    'Оля',
+    'Гурген',
+  ]
+};
+
+var getRandomIntInclusive = function (min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+};
+
+var createMockPicturesData = function (numberOfPictures) {
+  var mockObjects = [];
+
+  for (var i = 1; i < numberOfPictures + 1; i++) {
+    var mockComments = [];
+
+    for (var j = 0; j < getRandomIntInclusive(1, 5); j++) {
+      var mockComment = {
+        avatar: 'img/avatar-' + getRandomIntInclusive(1, 6) + '.svg',
+        message: MockData.COMMENTS_MESSAGES[Math.floor(Math.random() * MockData.COMMENTS_MESSAGES.length)],
```

Comment:
У тебя уже есть функция рандомизации.
```suggestion
        message: MockData.COMMENTS_MESSAGES[getRandomIntInclusive(0, MockData.COMMENTS_MESSAGES.lenght - 1)],
```

А ещё лучше - напиши функцию `getRandomArrayElement`, которая принимает массив а возвращает случайный элемент массива.